### DISCUSSION TITLE
How use SRT reader and publisher with ffmepg

### DISCUSSION THREAD

**Discussion by calanor** [bluenviron/mediamtx#2285] [General]
Hello,
I testing the new SRT feature and work fine if I use a srt-live-transmit application, but if I use a ffmpeg I get an Input/Output error. 

Reader example:

`srt-live-transmit srt://localhost:8890?streamid=read:testsrt udp://@239.192.1.1:1234  ` 
work fine.

`ffmpeg -i  srt://localhost:8890?streamid=read:testsrt  -c copy -f mpegts udp://@239.192.1.1:1234`
Error:      _srt://localhost:8890?streamid=read:testsrt: Input/output error_

Its possible to use an SRT feature with ffmpeg?
(also for publisher)

Thanks

**Comment by aler9** [comment]
Hello, FFmpeg with SRT works without issues, but here it's impossible to understand what the issue is without any additional information (server logs / network dumps / anything). Please open an issue and follow instructions in order to provide needed data that allows to replicate / understand the issue.
